Code/GraphMol/MolStandardize/Metal.h

```diff
@@ -64,10 +78,21 @@ accordingly.
     std::vector<int> boundMetalIndices;
   };
   int chargeAdjustment(const Atom *a, int order);
-  ROMOL_SPTR metal_nof;
-  ROMOL_SPTR metal_non;
+  ROMOL_SPTR metal_nof_;
```

Comment:
I'd prefer not to use the `_` at the end.
Pleae just prefix data member names with `m_` and pointers with `mp_`. So this becomes `dp_metal_nof`